### DISCUSSION TITLE
handbook/getting-started/serenity-js-with-playwright-test/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#2556] [Comments]
# handbook/getting-started/serenity-js-with-playwright-test/

Serenity/JS offers excellent support for Playwright, including Playwright Test, UI Mode, trace viewer, and auto-waiting locators!

https://serenity-js.org/handbook/getting-started/serenity-js-with-playwright-test/

**Comment by raducybersec** [comment]
in file playwright.config.ts  there is an error in this snippet  : 
```
 use: {
        // Serenity/JS configuration options
        crew: [
            // Automatically take screenshots upon an assertion failure
            ['@serenity-js/web:Photographer', { strategy: 'TakePhotosOfFailures' }]
        ],
        defaultActorName: 'Tess',
    },

    // Other Playwright Test configuration options
}
 ```
 there is an error
 "Object literal may only specify known properties, and 'crew' does not exist in type 'UseOptions<PlaywrightTestOptions & CustomProperties<{}>, PlaywrightWorkerOptions & CustomProperties<{}>>'."

**Reply by jan-molak** [reply]
Thank you, well spotted! Playwright Test configuration API has changed since I wrote this article. 
The recommended way now is to use `defineConfig` with a type parameter describing any additional options:

```ts
import { defineConfig, devices } from '@playwright/test'
import type { SerenityOptions } from '@serenity-js/playwright-test'

export default defineConfig<SerenityOptions>({
 // ...
})
```

I updated the tutorial to reflect that :+1:
